wordpress-mobile/WordPress-Android
# Feature/unreplied comments filter

This PR adds an Unreplied comments filter:
[![Image from Gyazo](https://i.gyazo.com/c12548882921f88bee35c16afcb73104.png)](https://gyazo.com/c12548882921f88bee35c16afcb73104)

[Here](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1937) is the companion Flux-C PR.

Unreplied filter shows recent comments that are:
- Top level (not a reply to other comment)
- Not made by a logged in user.
- Do not have a reply from logged in user.
- Is Approved or Pending.

Unreplied filter has no paging, and only shows unreplied comments from 100 most recent comments.

To test:

Comment structure you are looking to test looks like this:
[![Image from Gyazo](https://i.gyazo.com/0653937d52dd0c4b5d19f1afbc91164a.png)](https://gyazo.com/0653937d52dd0c4b5d19f1afbc91164a)

- Navigate to Comments List and select an Unreplied filter.
- Make sure that only comments that meet following conditions are visible
  - Top Level (direct reply to post, and not to another comment).
  - Not made by logged in user.
  - Do not have a reply from logged in user.
  - Is Approved or Pending.
- Open an Unreplied comment and reply to it.
- Navigate back to Comments List, and confirm that the comment you replied too is no longer in Unreplied comments list.

PR submission checklist:
